libpromises/evalfunction.c

```diff
@@ -2775,6 +2847,22 @@ static FnCallResult FnCallMapData(EvalContext *ctx, ARG_UNUSED const Policy *pol
         return FnFailure();
     }
 
+    if (mapdatamode && jqmode)
+    {
+        // TODO: make this customizable
```

Comment:
Hmmm you are right. Actually it would be preferable if we were only executing "jq" and having the system look it up in `PATH`. Unfortunately `cf_popen_full_duplex()` uses `execv()`, what I think would make sense would be to use `execvp()` instead. Quoting the man page:

> The execlp(), execvp(), and execvpe() functions duplicate the actions of the shell in searching for an executable file if the specified filename does not contain a slash (/) character.

Maybe change `cf_popen_full_duplex()` or duplicate it in an alternative? Do you think this would cover most customization issues?